packages/algob/test/lib/tx.ts

```diff
@@ -774,7 +774,7 @@ describe("Update transaction test in run mode", () => {
       type: wtypes.TransactionType.UpdateApp,
       sign: wtypes.SignType.SecretKey,
       fromAccount: bobAcc,
-      appID: appInfo?.appID,
+      appID: appInfo?.appID as number,
```

Comment:
I think linter will wipe this out. you're pushing with `--no-verify`. You should use `if (appInfo !== undefined) { // do stuff }`